files/usr/bin/system-flatpak-setup

```diff
@@ -60,7 +60,7 @@ if [[ -f $INSTALL_LIST_FILE ]]; then
   fi
   if [[ -n $INSTALL_LIST ]]; then
     flatpak install --system --noninteractive "$REPO_NAME" ${INSTALL_LIST[@]}
-    if [[ $NOTIFICATIONS == 1 ]]; then
+    if [[ $NOTIFICATIONS == "$true" ]]; then
```

Comment:
And here, what does the `$` here mean/do? Just to make sure I'm not missing something obvious I set `NOTIFICATIONS` to `true` in a local bash shell and then tried to run this if-statement, and the notification wasn't sent. And when I removed the dollar sign it was...

Reply:
quotation marks treat `$true` as a single string instead of `true`, again, my mistake.

Had similar mistake with quoting "${INSTALL_LIST[@]}" f.e.